README.md

```diff
@@ -9,12 +9,12 @@ This isn't the official client for LMS.
 Run this in terminal.
 
 ```bash
-wget -qO- https://raw.githubusercontent.com/GertJan84/lms-rust-cli/main/install | python
+wget -qO lms https://github.com/gertjan84/lms-rust-cli/releases/latest/download/lms && chmod +x lms && ./lms install
```

Comment:
`lms: Is a directory`

This command does not work, looks like wrong linux directory trickery